widgetssdk/src/main/java/com/glia/widgets/core/engagement/domain/IsOngoingEngagementUseCase.kt

```diff
@@ -1,9 +1,15 @@
 package com.glia.widgets.core.engagement.domain
 
 import com.glia.widgets.core.engagement.GliaEngagementRepository
+import com.glia.widgets.core.queue.GliaQueueRepository
+import com.glia.widgets.core.queue.model.GliaQueueingState
 
-class IsOngoingEngagementUseCase(private val engagementRepository: GliaEngagementRepository) {
+class IsOngoingEngagementUseCase(
+    private val engagementRepository: GliaEngagementRepository,
+    private val queueRepository: GliaQueueRepository
+) {
     operator fun invoke(): Boolean {
         return engagementRepository.hasOngoingEngagement()
+                || queueRepository.queueingState !is GliaQueueingState.None
```

Comment:
I would suggest making a separate use case "IsQueueingEngagement" because to me it's logically different states. "hasOngoingEngagement" happens after `QuueTicket` is handled and `Engagement` is created, isn't it?

Reply:
fixed [a50015914](https://github.com/salemove/android-sdk-widgets/pull/454/commits/a500159142065924efd9f475e49aec22034976b1)